lms/services/course.py

```diff
@@ -72,16 +72,16 @@ def get_from_launch(self, product, lti_params):
             copied_from=historical_course,
         )
 
-    def search(  # noqa: PLR0913, PLR0917
+    def _search_query(  # noqa: PLR0913, PLR0917
```

Comment:
Make this method return a raw SQLAlchmey query, we use the same pattern in other services (eg Organizationservice).